planner/core/task.go

```diff
@@ -1146,27 +1168,26 @@ func (p *PhysicalHashAgg) cpuCostDivisor(hasDistinct bool) (float64, float64) {
 	return math.Min(float64(finalCon), float64(partialCon)), float64(finalCon + partialCon)
 }
 
-func isFlashCopTask(cop *copTask) bool {
+func getCopTaskType(cop *copTask) kv.StoreType {
```

Comment:
Prefer to define a method to `copTask`, e.g:
```go
func (cop *copTask) getCopTaskType() kv.StoreType {
```

Reply:
Great.